tests/unit/src/test/scala/tests/RenameSuite.scala

```diff
@@ -279,6 +279,29 @@ object RenameSuite extends BaseLspSuite("rename") {
       Map("a/src/main/scala/a/Main.scala" -> "a/src/main/scala/a/Tree.scala")
```

Comment:
Unrelated to this PR, but I just noticed that this test suite should be called `RenameLspSuite`! cc/ @tgodzik

Reply:
I can squeeze a fix in ;-)

Reply:
You can use rename for the renaming of RenameLspSuite :O

Reply:
🤯

Reply:
Done!

I've renamed the object using the rename command, forgot to rename the file... but "rename" had my back! 😍